docs/apis-clients/tasklist-api/mutations/claim-task.mdx

```diff
@@ -3,7 +3,7 @@ id: claim-task
 title: claimTask
 ---
 
-Claim a task with taskId to currently logged in user. Returns the task.
+Claim a task with taskId to currently logged in user. Returns the task.ß
```

Comment:
What does this addition symbolize? I may need to add to the style guide.

Reply:
Good catch. I think that was a weird leftover symbol when I removed the text. Not sure whre it came from or why it appeared. I Removed it.

Reply:
I was thinking "hm, I don't know that I've seen this in technical writing before but maybe it's common knowledge for beta or something?" 😆